acquisitions/src/main/resources/domain/mod-invoice/samples/invoice.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Please rename file to mod-invoice/samples/invoice/to-check-invoice-and-invoice-lines-deletion-restrictions.json